crates/stc_ts_file_analyzer/src/analyzer/types/mod.rs

```diff
@@ -1170,6 +1245,11 @@ impl Analyzer<'_, '_> {
             }
 
             Type::Enum(e) => self.enum_to_type_lit(e).map(Cow::Owned)?,
+            Type::EnumVariant(e) => match self.expand_enum_variant(stc_ts_types::Type::EnumVariant(e.clone())) {
```

Comment:
This logic should be behind a flag in `NormalizeTypeOpts`.

Reply:
I don't understand 

If you understand why you have to do that,
Faster work will be possible.

Reply:
Because `normalize`  **should not** convert enum variants to literals **by default**.
`normalize `is a function used everywhere, and many of them has a special rule for enum variants.

Reply:
I'm sorry.  
I erased this part, and it works fine.